app/views/layouts/application.html.erb

```diff
@@ -7,12 +7,10 @@
     <%= tag.meta name: "viewport", content: "width=device-width, initial-scale=1" %>
     <%= tag.meta property: "og:image", content: asset_path("images/govuk-opengraph-image.png") %>
     <%= tag.meta name: "theme-color", content: "#0b0c0c" %>
-    <%= favicon_link_tag asset_path("images/favicon.ico") %>
-    <%= favicon_link_tag asset_path("images/govuk-mask-icon.svg"), rel: "mask-icon", type: "image/svg", color: "#0b0c0c" %>
-    <%= favicon_link_tag asset_path("images/govuk-apple-touch-icon.png"), rel: "apple-touch-icon", type: "image/png" %>
-    <%= favicon_link_tag asset_path("images/govuk-apple-touch-icon-152x152.png"), rel: "apple-touch-icon", type: "image/png", size: "152x152" %>
-    <%= favicon_link_tag asset_path("images/govuk-apple-touch-icon-167x167.png"), rel: "apple-touch-icon", type: "image/png", size: "167x167" %>
-    <%= favicon_link_tag asset_path("images/govuk-apple-touch-icon-180x180.png"), rel: "apple-touch-icon", type: "image/png", size: "180x180" %>
+    <%= favicon_link_tag asset_path("images/favicon.ico"), type: nil, sizes: "48x48" %>
+    <%= favicon_link_tag asset_path("images/favicon.svg"), type: "image/svg+xml", sizes: "any" %>
+    <%= favicon_link_tag asset_path("images/govuk-icon-mask.svg"), rel: "mask-icon", color: "#0b0c0c", type: nil %>
+    <%= favicon_link_tag asset_path("images/govuk-icon-180.png"), rel: "apple-touch-icon", type: nil %>
```

Comment:
just checking this is right as we've only got one apple-touch-icon, great if so!

Reply:
Hmm, good point, I mean it would have been wrong before this change as well 🤔

Reply:
Okay, yeah, this needs fixing

Reply:
Okay, turns out, didn't need fixing. The old ones don't load on review apps because of the permissions, so we get some errors in the console, the same thing happens with these ones. 
But it loads fine in the tests unlike the old ones so we can be pretty confident that they're coming through from govuk-frontend as we expect